doc/man/salt.7

```diff
@@ -31022,7 +31022,7 @@ common_packages:
 .UNINDENT
 .UNINDENT
 .sp
-tranlastes to:
+translates to:
```

Comment:
In the future, you don't need to change the man pages. These are auto-generated at release time from the files in `doc/`. This is fine for now, just a friendly note. ;)